packages/frontend/app/components/ui/Input.tsx

```diff
@@ -38,6 +39,9 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
             {...props}
           />
         </div>
+        {description ? (
+          <div className='font-medium text-sm'>{description}</div>
```

Comment:
```suggestion
          <div className='font-light italic mt-1 text-sm'>{description}</div>
```
maybe? to give a bit more visual separation

![Screenshot 2024-05-13 at 19 09 39](https://github.com/interledger/rafiki/assets/15069181/30c9f329-cc73-4da8-8329-599583d92b5e)